### DISCUSSION TITLE
RAG Problem + Slow loading in the interface in V 4.0.4

### DISCUSSION THREAD

**Discussion by ahyaha** [open-webui/open-webui#7354] [Issues]
![image](https://github.com/user-attachments/assets/6ceb6b6e-1ca1-49e0-ab5b-f36bfccf7079)

Turn this off.

**Comment by ips972** [comment]
+1 - noticed that to.. very long load times.

**Reply by Akira-baa** [reply]
+1 I upgrade to v0.4.6 but it's the same

**Reply by vscv** [reply]
same as v0.4.8
